modeling strategy proposal from bmi, hgb, bw (#750)
ihmeuw/vivarium_research: docs/source/gbd2019_models/risk_risk_correlation/maternal_bmi_hgb_birthweight/index.rst

Maternal BMI, Maternal Hemoglobin, and Infant Birthweight Risk Correlation
=============================================================================

Risk Exposures Overview
------------------------

Maternal undernourishment, including correlated body mass index (BMI) and anemia exposures, is associated adverse child health outcomes, including infant birthweight.

Risk Exposures in GBD
-----------------------

Links to documentation for relevant risk exposure pages include:

- :ref:`GBD 2019 Low birthweight short gestation risk exposure <2019_risk_exposure_lbwsg>`

- :ref:`Maternal BMI risk exposure conditional on hemoglobin status <2019_risk_exposure_maternal_bmi_hgb>`

- :ref:`Hemoglobin exposure <2019_hemoglobin_model>`

Vivarium Modeling Strategy
----------------------------

Correlation
++++++++++++

.. note::

   The strategy for modeling these risk-risk correlations related was developed for the needs of the :ref:`Intravenous Iron Intervention simulation <2019_concept_model_vivarium_iv_iron>`. Different strategies may be more appropriate for different project needs and should be reevaluated when necessary.

> We will model an association between a joint categorical distribution of maternal BMI and hemoglobin with infant birthweight. The asssoication between maternal BMI/hemoglobin categories will be measured in continuous shifts in infant birthweight. The magnitude of these shifts will be informed from BMGF trial data, shown below:

Reviewer: ```suggestion
We will model an association between a joint categorical distribution of maternal BMI and hemoglobin with infant birthweight. The association between maternal BMI/hemoglobin categories will be measured in continuous shifts in infant birthweight. The magnitude of these shifts will be informed from BMGF trial data, shown below:
```

.. note::

   Modeling strategy may be adapted if BMGF data is available for proportion born with low BW (<2500g) only (rather than mean birthweight). Briefly, in this case, we will use the probability born low birth weight for each maternal BMI/hemoglobin category to sample from LBWSG exposure categories with BW<2500 grams or from categories with BW>=2500g rather than apply shifts to birthweight values sampled from the overall LBWSG exposure distribution.

.. list-table:: Maternal BMI/Hemoglobin category unadjusted birthweight shifts
   :header-rows: 1 

   *  - Category
      - Maternal BMI exposure
      - Maternal Hemoglobin exposure
      - Birthweight shift
      - Note
   *  - cat4
      - >=18.5
      - >=10
      - 0
      - TMREL/Reference group
   *  - cat3
      - <18.5
      - >=10
      - TBD
      - Relative to cat4
   *  - cat2
      - >=18.5
      - <10
      - TBD
      - Relative to cat4
   *  - cat1
      - <18.5
      - <10
      - TBD
      - Relative to cat4
   
Given that,

.. math::

   BW_\text{cat{1,2,3} - cat4} = \overline{BW}_\text{cat{1,2,3}} - \overline{BW}_\text{cat4}

.. math::

>    \overline{BW}_{GBD} = \sum_{cat}^{n} p_\text{cat} * \overline{BW}_\text{cat}

Reviewer: Are these lines being calculated with GBD data? I thought GBD did not have mean birth weights by BMI/hemoglobin categories.

My expectation would have been to calculate the shifts for each category relative to the population mean in the BMGF data and then apply the relevant shift to the simulant's birthweight exposure value (pulled from the GBD distribution).

Author: Yes, I see your point, thanks for your close review. I changed the "GBD" reference here to "population" to avoid confusion around this point and then will update the specifics when we get the structure of the data from BMGF and see exactly how we can apply it.

Then the difference between maternal BMI strata and the population mean from GBD can be represented as:

.. math::

   BW_\text{cat4 - GBD} = p_\text{cat1} * BW_\text{cat1 - cat4}
                           + p_\text{cat2} * BW_\text{cat2 - cat4}
                           + p_\text{cat3} * BW_\text{cat3 - cat4}

Where :math:`p_\text{cat}` represents the exposure prevalence of a given maternal BMI/hemoglobin category.

> Therefore, the association bewteen maternal BMI/hemoglobin joint risk exposure and birthweight risk exposure should be implemented according to the following steps:

Reviewer: ```suggestion
Therefore, the association between maternal BMI/hemoglobin joint risk exposure and birthweight risk exposure should be implemented according to the following steps:
```

#. Apply a shift to the assigned continuous birthweight exposure value from step 2 based on the assigned maternal BMI exposure such that:

.. math::

   BW_\text{i, shifted} = BW_\text{i, unshifted} + shift_\text{cat_i, adjusted}

Where,

.. list-table:: Adjusted birthweight shifts
   :header-rows: 1

   *  - Parameter
      - Adjusted shift
      - Note
   *  - :math:`shift_\text{cat4, adjusted}`
      - :math:`p_\text{cat1} * shift_\text{cat1, unadjusted} + p_\text{cat2} * shift_\text{cat2, unadjusted} + p_\text{cat3} * shift_\text{cat3, unadjusted}`
      - 
   *  - :math:`shift_\text{cat3, adjusted}`
      - :math:`shift_\text{cat4, adjusted} + shift_\text{cat3, unadjusted}`
      - 
   *  - :math:`shift_\text{cat2, adjusted}`
      - :math:`shift_\text{cat4, adjusted} + shift_\text{cat2, unadjusted}`
      - 
   *  - :math:`shift_\text{cat1, adjusted}`
      - :math:`shift_\text{cat4, adjusted} + shift_\text{cat1, unadjusted}`
      - 

.. note::

>    These LBWSG exposure values may be later modified by intervention coverage and/or other factors. Note that a shift in continuous LBWSG exposure values may cause a simulant's going LBWSG exposure values to no longer fall within a valid GBD LBWSG exposure category. However, relative risks for the shifted exposure can still be calculated according ot the :ref:`LBWSG risk effects modeling strategy <2019_risk_effect_lbwsg>`.

Reviewer: ```suggestion
   These LBWSG exposure values may be later modified by intervention coverage and/or other factors. Note that a shift in continuous LBWSG exposure values may cause a simulant's LBWSG exposure value to no longer fall within a valid GBD LBWSG exposure category. However, relative risks for the shifted exposure can still be calculated according to the :ref:`LBWSG risk effects modeling strategy <2019_risk_effect_lbwsg>`.
```

   The gestational age assigned to the mother should be used to determine the duration of her pregnancy.

   The gestational age and birtweight exposure values assigned to the mother should be used to determine the child's LBWSG exposure value and relative risks during the neonatal period.

Causation
++++++++++++

We are not currently modeling a direct causal relationship between changes in maternal BMI exposure and changes in birthweight exposure.

> Assumptions and Limitations

Reviewer: It seems like we assume that the shifts are additive (as opposed to multiplicative, for instance). Is that worth noting here?

Author: I added a point to this section regarding how we apply a population mean difference to individuals as an additive shift and the potential implications. Thanks for noting this!
